contracts/WooCrossChainRouter.sol

```diff
@@ -46,27 +52,28 @@ contract WooCrossChainRouter is IStargateReceiver, InitializableOwnable, Reentra
     IWooPP public wooPool;
     address public quoteToken;
     address public WETH;
-    uint256 public bridgeSlippage = 100; // 1 in 10000th: default 1%
+    uint256 public bridgeSlippage; // 1 in 10000th: default 1%
```

Comment:
upgradeable contract don't allow initialize variables here, must in `initialize()`